test/common/upstream/utility.h

```diff
@@ -123,6 +123,14 @@ makeLocalityWeights(std::initializer_list<uint32_t> locality_weights) {
   return std::make_shared<LocalityWeights>(locality_weights);
 }
 
+inline envoy::config::core::v3::HealthCheck
+parseHealthCheckFromV3Yaml(const std::string& yaml_string, bool avoid_boosting = false) {
```

Comment:
I would think we would want to avoid boosting by default, and only in a few cases, e.g. deprecated tests, allow boosting.